[1.031s] Invoking command in 'C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\src\my_package': PYTHONPATH=C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\prefix_override;c:\python37\lib\site-packages\colcon_core\task\python\colcon_distutils_commands;C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\install\my_package\Lib\site-packages;%PYTHONPATH% c:\python37\python.exe -W ignore:setup.py install is deprecated -W ignore:easy_install command is deprecated setup.py egg_info --egg-base ..\..\build\my_package build --build-base C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\build install --record C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\install.log --install-scripts $base/lib/my_package --single-version-externally-managed install_data
[1.344s] running egg_info
[1.344s] creating ..\..\build\my_package\my_package.egg-info
[1.359s] writing ..\..\build\my_package\my_package.egg-info\PKG-INFO
[1.359s] writing dependency_links to ..\..\build\my_package\my_package.egg-info\dependency_links.txt
[1.359s] writing entry points to ..\..\build\my_package\my_package.egg-info\entry_points.txt
[1.391s] writing requirements to ..\..\build\my_package\my_package.egg-info\requires.txt
[1.391s] writing top-level names to ..\..\build\my_package\my_package.egg-info\top_level.txt
[1.391s] writing manifest file '..\..\build\my_package\my_package.egg-info\SOURCES.txt'
[1.406s] reading manifest file '..\..\build\my_package\my_package.egg-info\SOURCES.txt'
[1.406s] writing manifest file '..\..\build\my_package\my_package.egg-info\SOURCES.txt'
[1.422s] running build
[1.422s] running build_py
[1.422s] creating C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\build
[1.422s] creating C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\build\lib
[1.422s] creating C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\build\lib\my_package
[1.422s] copying my_package\my_node.py -> C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\build\lib\my_package
[1.422s] copying my_package\__init__.py -> C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\build\lib\my_package
[1.422s] running install
[1.422s] running install_lib
[1.438s] c:\python37\lib\site-packages\setuptools\dist.py:745: SetuptoolsDeprecationWarning: Invalid dash-separated options
[1.438s] !!
[1.438s] 
[1.438s]         ********************************************************************************
[1.438s]         Usage of dash-separated 'script-dir' will not be supported in future
[1.438s]         versions. Please use the underscore name 'script_dir' instead.
[1.438s] 
[1.438s]         This deprecation is overdue, please update your project and remove deprecated
[1.438s]         calls to avoid build errors in the future.
[1.438s] 
[1.438s]         See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
[1.438s]         ********************************************************************************
[1.438s] 
[1.438s] !!
[1.438s]   opt = self.warn_dash_deprecation(opt, section)
[1.438s] --- Logging error ---
[1.438s] Traceback (most recent call last):
[1.438s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.438s]     stream.write(msg + self.terminator)
[1.438s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.438s] Call stack:
[1.438s]   File "setup.py", line 23, in <module>
[1.438s]     'my_node = my_package.my_node:main'
[1.438s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.438s]     return distutils.core.setup(**attrs)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.438s]     return run_commands(dist)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.438s]     dist.run_commands()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.438s]     self.run_command(cmd)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.438s]     super().run_command(command)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.438s]     cmd_obj.run()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.438s]     return orig.install.run(self)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.438s]     self.run_command(cmd_name)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.438s]     self.distribution.run_command(command)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.438s]     super().run_command(command)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.438s]     cmd_obj.run()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 12, in run
[1.438s]     outfiles = self.install()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 115, in install
[1.438s]     outfiles = self.copy_tree(self.build_dir, self.install_dir)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 95, in copy_tree
[1.438s]     return orig.install_lib.copy_tree(self, infile, outfile)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 379, in copy_tree
[1.438s]     dry_run=self.dry_run,
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 151, in copy_tree
[1.438s]     mkpath(dst, verbose=verbose)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.438s]     log.info("creating %s", head)
[1.438s] Message: 'creating %s'
[1.438s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package',)
[1.438s] --- Logging error ---
[1.438s] Traceback (most recent call last):
[1.438s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.438s]     stream.write(msg + self.terminator)
[1.438s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.438s] Call stack:
[1.438s]   File "setup.py", line 23, in <module>
[1.438s]     'my_node = my_package.my_node:main'
[1.438s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.438s]     return distutils.core.setup(**attrs)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.438s]     return run_commands(dist)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.438s]     dist.run_commands()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.438s]     self.run_command(cmd)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.438s]     super().run_command(command)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.438s]     cmd_obj.run()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.438s]     return orig.install.run(self)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.438s]     self.run_command(cmd_name)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.438s]     self.distribution.run_command(command)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.438s]     super().run_command(command)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.438s]     cmd_obj.run()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 12, in run
[1.438s]     outfiles = self.install()
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 115, in install
[1.438s]     outfiles = self.copy_tree(self.build_dir, self.install_dir)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 95, in copy_tree
[1.438s]     return orig.install_lib.copy_tree(self, infile, outfile)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 379, in copy_tree
[1.438s]     dry_run=self.dry_run,
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 151, in copy_tree
[1.438s]     mkpath(dst, verbose=verbose)
[1.438s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.438s]     log.info("creating %s", head)
[1.438s] Message: 'creating %s'
[1.438s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib',)
[1.438s] --- Logging error ---
[1.438s] Traceback (most recent call last):
[1.438s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.438s]     stream.write(msg + self.terminator)
[1.438s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.438s] Call stack:
[1.438s]   File "setup.py", line 23, in <module>
[1.438s]     'my_node = my_package.my_node:main'
[1.438s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.438s]     return distutils.core.setup(**attrs)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.453s]     return run_commands(dist)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.453s]     dist.run_commands()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.453s]     self.run_command(cmd)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.453s]     return orig.install.run(self)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.453s]     self.run_command(cmd_name)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.453s]     self.distribution.run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 12, in run
[1.453s]     outfiles = self.install()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 115, in install
[1.453s]     outfiles = self.copy_tree(self.build_dir, self.install_dir)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 95, in copy_tree
[1.453s]     return orig.install_lib.copy_tree(self, infile, outfile)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 379, in copy_tree
[1.453s]     dry_run=self.dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 151, in copy_tree
[1.453s]     mkpath(dst, verbose=verbose)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.453s]     log.info("creating %s", head)
[1.453s] Message: 'creating %s'
[1.453s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib\\site-packages',)
[1.453s] --- Logging error ---
[1.453s] Traceback (most recent call last):
[1.453s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.453s]     stream.write(msg + self.terminator)
[1.453s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.453s] Call stack:
[1.453s]   File "setup.py", line 23, in <module>
[1.453s]     'my_node = my_package.my_node:main'
[1.453s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.453s]     return distutils.core.setup(**attrs)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.453s]     return run_commands(dist)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.453s]     dist.run_commands()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.453s]     self.run_command(cmd)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.453s]     return orig.install.run(self)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.453s]     self.run_command(cmd_name)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.453s]     self.distribution.run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 12, in run
[1.453s]     outfiles = self.install()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 115, in install
[1.453s]     outfiles = self.copy_tree(self.build_dir, self.install_dir)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 95, in copy_tree
[1.453s]     return orig.install_lib.copy_tree(self, infile, outfile)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 379, in copy_tree
[1.453s]     dry_run=self.dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 181, in copy_tree
[1.453s]     dry_run=dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 151, in copy_tree
[1.453s]     mkpath(dst, verbose=verbose)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.453s]     log.info("creating %s", head)
[1.453s] Message: 'creating %s'
[1.453s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib\\site-packages\\my_package',)
[1.453s] --- Logging error ---
[1.453s] Traceback (most recent call last):
[1.453s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.453s]     stream.write(msg + self.terminator)
[1.453s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 153: illegal multibyte sequence
[1.453s] Call stack:
[1.453s]   File "setup.py", line 23, in <module>
[1.453s]     'my_node = my_package.my_node:main'
[1.453s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.453s]     return distutils.core.setup(**attrs)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.453s]     return run_commands(dist)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.453s]     dist.run_commands()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.453s]     self.run_command(cmd)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.453s]     return orig.install.run(self)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.453s]     self.run_command(cmd_name)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.453s]     self.distribution.run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 12, in run
[1.453s]     outfiles = self.install()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 115, in install
[1.453s]     outfiles = self.copy_tree(self.build_dir, self.install_dir)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 95, in copy_tree
[1.453s]     return orig.install_lib.copy_tree(self, infile, outfile)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 379, in copy_tree
[1.453s]     dry_run=self.dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 181, in copy_tree
[1.453s]     dry_run=dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 192, in copy_tree
[1.453s]     dry_run=dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\file_util.py", line 137, in copy_file
[1.453s]     log.info("%s %s -> %s", action, src, dir)
[1.453s] Message: '%s %s -> %s'
[1.453s] Arguments: ('copying', 'C:\\Users\\SSAFY\\Desktop\\doyun\\Ưȭ\\S12P21A703\\sub1\\doyun\\catkin_ws\\build\\my_package\\build\\lib\\my_package\\my_node.py', 'C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib\\site-packages\\my_package')
[1.453s] --- Logging error ---
[1.453s] Traceback (most recent call last):
[1.453s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.453s]     stream.write(msg + self.terminator)
[1.453s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 154: illegal multibyte sequence
[1.453s] Call stack:
[1.453s]   File "setup.py", line 23, in <module>
[1.453s]     'my_node = my_package.my_node:main'
[1.453s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.453s]     return distutils.core.setup(**attrs)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.453s]     return run_commands(dist)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.453s]     dist.run_commands()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.453s]     self.run_command(cmd)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.453s]     return orig.install.run(self)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.453s]     self.run_command(cmd_name)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.453s]     self.distribution.run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 12, in run
[1.453s]     outfiles = self.install()
[1.453s] running install_data
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 115, in install
[1.453s]     outfiles = self.copy_tree(self.build_dir, self.install_dir)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 95, in copy_tree
[1.453s]     return orig.install_lib.copy_tree(self, infile, outfile)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 379, in copy_tree
[1.453s]     dry_run=self.dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 181, in copy_tree
[1.453s]     dry_run=dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 192, in copy_tree
[1.453s]     dry_run=dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\file_util.py", line 137, in copy_file
[1.453s]     log.info("%s %s -> %s", action, src, dir)
[1.453s] Message: '%s %s -> %s'
[1.453s] Arguments: ('copying', 'C:\\Users\\SSAFY\\Desktop\\doyun\\Ưȭ\\S12P21A703\\sub1\\doyun\\catkin_ws\\build\\my_package\\build\\lib\\my_package\\__init__.py', 'C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib\\site-packages\\my_package')
[1.453s] --- Logging error ---
[1.453s] Traceback (most recent call last):
[1.453s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.453s]     stream.write(msg + self.terminator)
[1.453s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 44: illegal multibyte sequence
[1.453s] Call stack:
[1.453s]   File "setup.py", line 23, in <module>
[1.453s]     'my_node = my_package.my_node:main'
[1.453s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.453s]     return distutils.core.setup(**attrs)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.453s]     return run_commands(dist)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.453s]     dist.run_commands()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.453s]     self.run_command(cmd)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.453s]     return orig.install.run(self)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.453s]     self.run_command(cmd_name)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.453s]     self.distribution.run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 15, in run
[1.453s]     self.byte_compile(outfiles)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 142, in byte_compile
[1.453s]     dry_run=self.dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\util.py", line 500, in byte_compile
[1.453s]     log.info("byte-compiling %s to %s", file, cfile_base)
[1.453s] Message: 'byte-compiling %s to %s'
[1.453s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib\\site-packages\\my_package\\my_node.py', 'my_node.cpython-37.pyc')
[1.453s] --- Logging error ---
[1.453s] Traceback (most recent call last):
[1.453s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.453s]     stream.write(msg + self.terminator)
[1.453s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 44: illegal multibyte sequence
[1.453s] Call stack:
[1.453s]   File "setup.py", line 23, in <module>
[1.453s]     'my_node = my_package.my_node:main'
[1.453s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.453s]     return distutils.core.setup(**attrs)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.453s]     return run_commands(dist)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.453s]     dist.run_commands()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.453s]     self.run_command(cmd)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.453s]     return orig.install.run(self)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.453s]     self.run_command(cmd_name)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.453s]     self.distribution.run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.453s]     super().run_command(command)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.453s]     cmd_obj.run()
[1.453s]   File "c:\python37\lib\site-packages\setuptools\command\install_lib.py", line 15, in run
[1.453s]     self.byte_compile(outfiles)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_lib.py", line 142, in byte_compile
[1.453s]     dry_run=self.dry_run,
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\util.py", line 500, in byte_compile
[1.453s]     log.info("byte-compiling %s to %s", file, cfile_base)
[1.453s] Message: 'byte-compiling %s to %s'
[1.453s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib\\site-packages\\my_package\\__init__.py', '__init__.cpython-37.pyc')
[1.453s] --- Logging error ---
[1.453s] Traceback (most recent call last):
[1.453s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.453s]     stream.write(msg + self.terminator)
[1.453s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.453s] Call stack:
[1.453s]   File "setup.py", line 23, in <module>
[1.453s]     'my_node = my_package.my_node:main'
[1.453s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.453s]     return distutils.core.setup(**attrs)
[1.453s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.453s]     return run_commands(dist)
[1.469s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.469s]     dist.run_commands()
[1.469s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.469s]     self.run_command(cmd)
[1.469s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.469s]     super().run_command(command)
[1.469s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.469s]     cmd_obj.run()
[1.469s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.469s]     return orig.install.run(self)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.484s]     self.run_command(cmd_name)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.484s]     self.distribution.run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_data.py", line 65, in run
[1.484s]     self.mkpath(dir)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 342, in mkpath
[1.484s]     dir_util.mkpath(name, mode, dry_run=self.dry_run)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.484s]     log.info("creating %s", head)
[1.484s] Message: 'creating %s'
[1.484s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\share',)
[1.484s] --- Logging error ---
[1.484s] Traceback (most recent call last):
[1.484s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.484s]     stream.write(msg + self.terminator)
[1.484s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.484s] Call stack:
[1.484s]   File "setup.py", line 23, in <module>
[1.484s]     'my_node = my_package.my_node:main'
[1.484s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.484s]     return distutils.core.setup(**attrs)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.484s]     return run_commands(dist)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.484s]     dist.run_commands()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.484s]     self.run_command(cmd)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.484s]     return orig.install.run(self)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.484s]     self.run_command(cmd_name)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.484s]     self.distribution.run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_data.py", line 65, in run
[1.484s]     self.mkpath(dir)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 342, in mkpath
[1.484s]     dir_util.mkpath(name, mode, dry_run=self.dry_run)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.484s]     log.info("creating %s", head)
[1.484s] Message: 'creating %s'
[1.484s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\share\\ament_index',)
[1.484s] --- Logging error ---
[1.484s] Traceback (most recent call last):
[1.484s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.484s]     stream.write(msg + self.terminator)
[1.484s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.484s] Call stack:
[1.484s]   File "setup.py", line 23, in <module>
[1.484s]     'my_node = my_package.my_node:main'
[1.484s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.484s]     return distutils.core.setup(**attrs)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.484s]     return run_commands(dist)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.484s]     dist.run_commands()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.484s]     self.run_command(cmd)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.484s]     return orig.install.run(self)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.484s]     self.run_command(cmd_name)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.484s]     self.distribution.run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_data.py", line 65, in run
[1.484s]     self.mkpath(dir)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 342, in mkpath
[1.484s]     dir_util.mkpath(name, mode, dry_run=self.dry_run)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.484s]     log.info("creating %s", head)
[1.484s] Message: 'creating %s'
[1.484s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\share\\ament_index\\resource_index',)
[1.484s] --- Logging error ---
[1.484s] Traceback (most recent call last):
[1.484s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.484s]     stream.write(msg + self.terminator)
[1.484s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.484s] Call stack:
[1.484s]   File "setup.py", line 23, in <module>
[1.484s]     'my_node = my_package.my_node:main'
[1.484s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.484s]     return distutils.core.setup(**attrs)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.484s]     return run_commands(dist)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.484s]     dist.run_commands()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.484s]     self.run_command(cmd)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.484s]     return orig.install.run(self)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.484s]     self.run_command(cmd_name)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.484s]     self.distribution.run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_data.py", line 65, in run
[1.484s]     self.mkpath(dir)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 342, in mkpath
[1.484s]     dir_util.mkpath(name, mode, dry_run=self.dry_run)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.484s]     log.info("creating %s", head)
[1.484s] Message: 'creating %s'
[1.484s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\share\\ament_index\\resource_index\\packages',)
[1.484s] --- Logging error ---
[1.484s] Traceback (most recent call last):
[1.484s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.484s]     stream.write(msg + self.terminator)
[1.484s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 60: illegal multibyte sequence
[1.484s] Call stack:
[1.484s]   File "setup.py", line 23, in <module>
[1.484s]     'my_node = my_package.my_node:main'
[1.484s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.484s]     return distutils.core.setup(**attrs)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.484s]     return run_commands(dist)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.484s]     dist.run_commands()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.484s]     self.run_command(cmd)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.484s]     super().run_command(command)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.484s]     cmd_obj.run()
[1.484s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.484s]     return orig.install.run(self)
[1.484s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.484s]     self.run_command(cmd_name)
[1.484s] running install_egg_info
[1.484s] running install_scripts
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.656s]     self.distribution.run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_data.py", line 76, in run
[1.656s]     (out, _) = self.copy_file(data, dir)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 357, in copy_file
[1.656s]     dry_run=self.dry_run,
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\file_util.py", line 137, in copy_file
[1.656s]     log.info("%s %s -> %s", action, src, dir)
[1.656s] Message: '%s %s -> %s'
[1.656s] Arguments: ('copying', 'resource\\my_package', 'C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\share\\ament_index\\resource_index\\packages')
[1.656s] --- Logging error ---
[1.656s] Traceback (most recent call last):
[1.656s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.656s]     stream.write(msg + self.terminator)
[1.656s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 38: illegal multibyte sequence
[1.656s] Call stack:
[1.656s]   File "setup.py", line 23, in <module>
[1.656s]     'my_node = my_package.my_node:main'
[1.656s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.656s]     return distutils.core.setup(**attrs)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.656s]     return run_commands(dist)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.656s]     dist.run_commands()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.656s]     self.run_command(cmd)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.656s]     return orig.install.run(self)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.656s]     self.run_command(cmd_name)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.656s]     self.distribution.run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_data.py", line 65, in run
[1.656s]     self.mkpath(dir)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 342, in mkpath
[1.656s]     dir_util.mkpath(name, mode, dry_run=self.dry_run)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dir_util.py", line 71, in mkpath
[1.656s]     log.info("creating %s", head)
[1.656s] Message: 'creating %s'
[1.656s] Arguments: ('C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\share\\my_package',)
[1.656s] --- Logging error ---
[1.656s] Traceback (most recent call last):
[1.656s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.656s]     stream.write(msg + self.terminator)
[1.656s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 52: illegal multibyte sequence
[1.656s] Call stack:
[1.656s]   File "setup.py", line 23, in <module>
[1.656s]     'my_node = my_package.my_node:main'
[1.656s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.656s]     return distutils.core.setup(**attrs)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.656s]     return run_commands(dist)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.656s]     dist.run_commands()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.656s]     self.run_command(cmd)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.656s]     return orig.install.run(self)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.656s]     self.run_command(cmd_name)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.656s]     self.distribution.run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install_data.py", line 76, in run
[1.656s]     (out, _) = self.copy_file(data, dir)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 357, in copy_file
[1.656s]     dry_run=self.dry_run,
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\file_util.py", line 137, in copy_file
[1.656s]     log.info("%s %s -> %s", action, src, dir)
[1.656s] Message: '%s %s -> %s'
[1.656s] Arguments: ('copying', 'package.xml', 'C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\share\\my_package')
[1.656s] --- Logging error ---
[1.656s] Traceback (most recent call last):
[1.656s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.656s]     stream.write(msg + self.terminator)
[1.656s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 83: illegal multibyte sequence
[1.656s] Call stack:
[1.656s]   File "setup.py", line 23, in <module>
[1.656s]     'my_node = my_package.my_node:main'
[1.656s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.656s]     return distutils.core.setup(**attrs)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.656s]     return run_commands(dist)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.656s]     dist.run_commands()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.656s]     self.run_command(cmd)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.656s]     return orig.install.run(self)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.656s]     self.run_command(cmd_name)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.656s]     self.distribution.run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\command\install_egg_info.py", line 40, in run
[1.656s]     self.copytree, (), "Copying %s to %s" % (self.source, self.target)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 339, in execute
[1.656s]     util.execute(func, args, msg, dry_run=self.dry_run)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\util.py", line 335, in execute
[1.656s]     log.info(msg)
[1.656s] Message: 'Copying ..\\..\\build\\my_package\\my_package.egg-info to C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\Lib\\site-packages\\my_package-0.0.0-py3.7.egg-info'
[1.656s] Arguments: ()
[1.656s] --- Logging error ---
[1.656s] Traceback (most recent call last):
[1.656s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.656s]     stream.write(msg + self.terminator)
[1.656s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 68: illegal multibyte sequence
[1.656s] writing list of installed files to 'C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\install.log'
[1.656s] Call stack:
[1.656s]   File "setup.py", line 23, in <module>
[1.656s]     'my_node = my_package.my_node:main'
[1.656s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.656s]     return distutils.core.setup(**attrs)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.656s]     return run_commands(dist)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.656s]     dist.run_commands()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.656s]     self.run_command(cmd)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.656s]     cmd_obj.run()
[1.656s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.656s]     return orig.install.run(self)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.656s]     self.run_command(cmd_name)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.656s]     self.distribution.run_command(command)
[1.656s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.656s]     super().run_command(command)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.672s]     cmd_obj.run()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install_scripts.py", line 26, in run
[1.672s]     self._install_ep_scripts()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install_scripts.py", line 57, in _install_ep_scripts
[1.672s]     self.write_script(*args)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install_scripts.py", line 63, in write_script
[1.672s]     log.info("Installing %s script to %s", script_name, self.install_dir)
[1.672s] Message: 'Installing %s script to %s'
[1.672s] Arguments: ('my_node-script.py', 'C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\lib\\my_package')
[1.672s] --- Logging error ---
[1.672s] Traceback (most recent call last):
[1.672s]   File "c:\python37\lib\logging\__init__.py", line 1028, in emit
[1.672s]     stream.write(msg + self.terminator)
[1.672s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 62: illegal multibyte sequence
[1.672s] Call stack:
[1.672s]   File "setup.py", line 23, in <module>
[1.672s]     'my_node = my_package.my_node:main'
[1.672s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.672s]     return distutils.core.setup(**attrs)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.672s]     return run_commands(dist)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.672s]     dist.run_commands()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.672s]     self.run_command(cmd)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.672s]     super().run_command(command)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.672s]     cmd_obj.run()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.672s]     return orig.install.run(self)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 708, in run
[1.672s]     self.run_command(cmd_name)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
[1.672s]     self.distribution.run_command(command)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.672s]     super().run_command(command)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.672s]     cmd_obj.run()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install_scripts.py", line 26, in run
[1.672s]     self._install_ep_scripts()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install_scripts.py", line 57, in _install_ep_scripts
[1.672s]     self.write_script(*args)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install_scripts.py", line 63, in write_script
[1.672s]     log.info("Installing %s script to %s", script_name, self.install_dir)
[1.672s] Message: 'Installing %s script to %s'
[1.672s] Arguments: ('my_node.exe', 'C:\\Users\\SSAFY\\Desktop\\doyun\\\u01af\u022d\\S12P21A703\\sub1\\doyun\\catkin_ws\\install\\my_package\\lib\\my_package')
[1.672s] Traceback (most recent call last):
[1.672s]   File "setup.py", line 23, in <module>
[1.672s]     'my_node = my_package.my_node:main'
[1.672s]   File "c:\python37\lib\site-packages\setuptools\__init__.py", line 107, in setup
[1.672s]     return distutils.core.setup(**attrs)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
[1.672s]     return run_commands(dist)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
[1.672s]     dist.run_commands()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
[1.672s]     self.run_command(cmd)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\dist.py", line 1234, in run_command
[1.672s]     super().run_command(command)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
[1.672s]     cmd_obj.run()
[1.672s]   File "c:\python37\lib\site-packages\setuptools\command\install.py", line 74, in run
[1.672s]     return orig.install.run(self)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\command\install.py", line 723, in run
[1.672s]     "writing list of installed files to '%s'" % self.record,
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\cmd.py", line 339, in execute
[1.672s]     util.execute(func, args, msg, dry_run=self.dry_run)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\util.py", line 337, in execute
[1.672s]     func(*args)
[1.672s]   File "c:\python37\lib\site-packages\setuptools\_distutils\file_util.py", line 246, in write_file
[1.672s]     f.write(line + "\n")
[1.672s] UnicodeEncodeError: 'cp949' codec can't encode character '\u01af' in position 29: illegal multibyte sequence
[1.672s] Invoked command in 'C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\src\my_package' returned '1': PYTHONPATH=C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\prefix_override;c:\python37\lib\site-packages\colcon_core\task\python\colcon_distutils_commands;C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\install\my_package\Lib\site-packages;%PYTHONPATH% c:\python37\python.exe -W ignore:setup.py install is deprecated -W ignore:easy_install command is deprecated setup.py egg_info --egg-base ..\..\build\my_package build --build-base C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\build install --record C:\Users\SSAFY\Desktop\doyun\Ưȭ\S12P21A703\sub1\doyun\catkin_ws\build\my_package\install.log --install-scripts $base/lib/my_package --single-version-externally-managed install_data
